server/webhook.go

```diff
@@ -75,7 +75,14 @@ func (wh webhook) PostToChannel(p *Plugin, channelId, fromUserId string) (*model
 	return post, http.StatusOK, nil
 }
 
-func (wh *webhook) PostNotifications(p *Plugin, ji Instance) ([]*model.Post, int, error) {
+func (wh *webhook) PostNotifications(p *Plugin) ([]*model.Post, int, error) {
+	// We will only send webhook events if we have a connected instance.
+	ji, err := p.currentInstanceStore.LoadCurrentJIRAInstance()
```

Comment:
Maybe move this after the check for len(notifications)? Also, we should probably return http.StatusOK there too...

Reply:
Done! And we already did return a 200 there.